app/controllers/users_controller.rb

```diff
@@ -126,7 +126,8 @@ def quick_new
     if ldap_result.nil?
       @message = 'Sorry, the login that you entered does not exist.
       You cannot create a user profile without a valid login.'
-      render(:quick_new) && return
+      render action: 'quick_new'
+      return
     end
 
     # Is there a user record already?
```

Comment:
The line below this should be changed from a guard clause to an `if` clause with the `@message = ` line within it.